docs/source/quasi_dags.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>Optional: But you might want to mention that IV and RCT allow unconfounded inference for slightly different estimands. You can refer to the paper for details, but you might also want to distinguish ATE, ATT, ATC, and LATE kinds of estimands and how they relate to different DAGs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>+1</p><p> </p><p>And maybe comment about the need of a 'strong' instrument. Becausee a weak relationship IV -&gt; Z can lead to high variance (right?)</p><p> </p><p>Also, add an example. My favorite on is <a href="https://matheusfacure.github.io/python-causality-handbook/09-Non-Compliance-and-LATE.html" target="_blank">https://matheusfacure.github.io/python-causality-handbook/09-Non-Compliance-and-LATE.html</a> Maybee we can borrow it (with the proper citation)</p>

Reply:
<p>I think this is all fair enough. For the moment I've bundled this points into <a href="https://github.com/pymc-labs/CausalPy/issues/327" target="_blank">https://github.com/pymc-labs/CausalPy/issues/327</a> so that they are definitely acted on. But feel free to let me know if you think these points need to be addressed in the first iteration.</p>

Reply:
I think we can merge this one and then create an issue to add examples.

Reply:
Agreed